example/zip/zip.js

```diff
@@ -3,353 +3,84 @@ node example/zip/zip.js
 */
 // this could be simplified
 var zip = require('../../src/zip'),
-  fs = require('fs');
+    fs = require('fs');
+
+var zipOptions = {
+    types: {
+        point: 'point',
+        pointz: 'pointz',
+        polyline: 'polyline',
+        polylinez: 'polylinez',
+        polygon: 'polygon',
+        polygonz: 'polygonz',
+    },
+};
+
+// utility
+function geojsonZipHelper(geojson, fileName) {
+    zip(geojson, zipOptions).then((base64String) => {
+        fs.writeFile(fileName, base64String, { encoding: 'base64' }, function (
+            err,
+        ) {
+            if (err) {
+                console.log('Error', err);
+            } else {
+                console.log('File created');
```

Comment:
`${fileName} created` would look better when executing this file.

![image](https://user-images.githubusercontent.com/8754/98041570-fffe6100-1dd6-11eb-8c21-a5554a492564.png)